js/script.js

```diff
@@ -8,3 +8,113 @@ document.querySelectorAll('.navbar__menu__link').forEach((n) => n.addEventListen
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Facebook 360',
+    info: ['facebook', 'backend', '2015'],
+    description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s. <span class='description-3'> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt animi consequuntur consectetur voluptate accusantium facilis, fuga non minus, </span>",
+    skills: ['html', 'css', 'javascript'],
+    image: 'images/desktop/facebook-360-project.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Uber Navigation',
+    info: ['UBER', ' Lead Developer', '2015'],
+    description:
+    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s. <span class='description-3'> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt animi consequuntur consectetur voluptate accusantium facilis, fuga non minus, </span>",
+    skills: ['html', 'css', 'javascript'],
+    image: 'images/desktop/uber-navigation-project.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+];
+
+const btns = document.querySelectorAll('.works button');
+const bgModal = document.querySelector('.bg-modal');
+
+function popUp(index) {
+  const {
+    title, info, description, skills, image,
+  } = cards[index];
+  const [info1, info2, info3] = info;
+  const [skill1, skill2, skill3] = skills;
+
+  bgModal.innerHTML = `
+  <div class="modal-content">
+        <div class="modal-header">
+          <h3 class="tonic">${title}</h3>
+          <span class="close-button">+</span>
+        </div>
+        <ul class="ul-box1 flex0">
+          <li class="canopy">${info1}</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">${info2}</li>
+          <li><img src="images/Counter.png" alt="counter" /></li>
+          <li class="light">${info3}</li>
+        </ul>
+        <a href="#"
+          ><img src="${image}" alt="Snapshoot-Portfolio" class="snapshot"
+        /></a>
+        <div class="flex-99">
+          <div class="flex-98">
+            <p class="description-2 flex0">
+            ${description}
+            </p>
+          </div>
+          <div>
+            <ul class="ul-box2 flex0">
+              <li class="highlight font-1">${skill1}</li>
+              <li class="highlight font-2">${skill2}</li>
+              <li class="highlight font-3">${skill3}</li>
+            </ul>
+            <hr class="solid-1">
+            <div class="buttons">
+              <button type="button" class="live-btn">
+                see live
+                <i class='fas fa-check'></i>
+              </button>
+              <button type="button" class="live-btn">
+                see source
+                <i class='fab fa-github'></i>
```

Comment:
_The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins)._

- [ ] It appears there are some styling issues that need to be addressed. Please rectify them accordingly.



![Screenshot 2022-12-22 at 01 37 52](https://user-images.githubusercontent.com/69011963/209000096-dc667b25-f414-483f-ae82-119d691e2228.png)

1. Please adjust the X button slightly upwards 🔼 
2. Add more spacing between facebook, backend and 2015 and the image ➕ 
3. The image appears a bit small; please ensure it's as wide as specified 📏 
4. Position your skills further to the right, with wider borders 🔳
5. Lastly, add the GitHub logo into its scheduled place 🗂

Reply:
Thi is the screenshot:
<img width="686" alt="Screen Shot 2022-12-22 at 9 57 38 PM" src="https://user-images.githubusercontent.com/4833083/209224850-e6a088d9-69a8-4fab-822f-b4b744d0e3e9.png">

<img width="932" alt="Screen Shot 2022-12-22 at 9 59 14 PM" src="https://user-images.githubusercontent.com/4833083/209225045-0130bea8-4377-446c-8039-554f04db17f7.png">

Reply:
are not you done yet? am seeing changes while am reviewing, please finish all and let me review the work @jugosack 

- [ ] I was marking this comment but you were rapid to add a comment before I even start, next time make sure you finish everything before you request for review, this is to say, still, in this section, I will raise REQUIRED CHANGE, please take a look at this image below and guess what is happening.

THIS IS THE FIGMA

- I tried to surround the image read with the red border

<img width="543" alt="1:" src="https://user-images.githubusercontent.com/77843050/209228725-d557ade1-3801-44bd-a170-7a16747cd8c7.png">


THIS IS YOURS

<img width="1280" alt="Screen Shot 2022-12-23 at 00 28 59" src="https://user-images.githubusercontent.com/77843050/209229649-8cc90577-affb-4718-b1f9-79f53790c300.png">

- [ ] All the changes and points from the previous reviewer said to be changed are still the same, you can compare this image with the one of the previous reviewer and the original, please fix it


- [ ] - [optional ] Time to raise an optional but important issue you should fix, let's jump on this record below and guess what is happening

![5](https://user-images.githubusercontent.com/77843050/209230347-e5f2d181-4954-4886-b690-331d8adaf577.gif)

- You have seen that? yeah I think, the window under the hero section when we scroll, I would suggest you add a z-index of 1 to fix this issue